soc/intel/intel_socfpga/agilex/Kconfig.soc

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2021-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_AGILEX
+	bool
+	help
+	  "Intel SoC FPGA Agilex Series"
+
+config SOC_SERIES
+	default "agilex" if SOC_SERIES_AGILEX
+
+config SOC_AGILEX
+	bool
+	help
+	  "Intel SoC FPGA Agilex"
+
+config SOC
+	default "intel_socfpga_agilex" if SOC_AGILEX
```

Comment:
```suggestion
	default "agilex" if SOC_AGILEX
```

Reply:
why a short name, just `agilex` ?

Reply:
Because SOC_AGILEX -> agilex

Reply:
I assume this note is also related to SoC/board name optimization as this discussion - https://github.com/zephyrproject-rtos/zephyr/pull/68915#discussion_r1487770683

In regard of this PR my opinion is the same: it is better to avoid ad-hoc name changes doing such a bulk migration unless there is naming convention for all the boards and SoC of a vendor aligned to the HWMv2 scheme.

For these SoC/boards It is possible to keep just the same board names as currently.
I would left to the board maintainers to decide when (now?/later?) and how exactly to change these names if it makes sense.